sensor/AdblockPlugin.js

```diff
@@ -89,6 +90,20 @@ class AdblockPlugin extends Sensor {
       extensionManager.onGet("adblockConfig", async (msg, data) => {
         return this.getAdblockConfig();
       });
+      extensionManager.onSet("adblockFastMode", async (msg, data) => {
```

Comment:
this api set is not necessary. policy:system already supports configuration load/change from app natively.

Refer to this example for api call to refresh adblock config, no need to send event ADBLOCK_CONFIG_REFRESH, directly call the apply change function
https://github.com/firewalla/firewalla/blob/master/sensor/FamilyProtectPlugin.js#L56